spec/features/editing_topics/edit_topics_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+RSpec.feature "Show all the topics" do
+  scenario do
+    given_there_is_a_document
+    when_i_visit_the_document_page
+    and_i_try_to_change_the_topics
+    then_i_see_all_of_the_topics
+  end
+
+  def given_there_is_a_document
+    create :document
+  end
+
+  def when_i_visit_the_document_page
+    visit document_path(Document.last)
+  end
+
+  def and_i_try_to_change_the_topics
+    # GOV.UK homepage
+    publishing_api_has_expanded_links(
+      "content_id" => "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a",
+      "expanded_links" => {
+        "level_one_taxons" => [
+          {
+            "title" => "Level One Topic",
+            "content_id" => "level_one_topic",
+          },
+        ],
+      },
+    )
+
+    publishing_api_has_expanded_links(
+      "content_id" => "level_one_topic",
+      "expanded_links" => {
+        "child_taxons" => [
+          {
+            "title" => "Level Two Topic",
+            "links" => {
+              "child_taxons" => [
+                {
+                  "title" => "Level Three Topic",
+                  "links" => {},
+                },
+              ],
+            },
+          },
+        ],
+      },
+    )
```

Comment:
This is pretty beastie. I think if we need to do this again for a different test we should create a helper.

Reply:
We think alike here :-). I thought I'd wait for further specs so we have more working examples.